build/common/installer/scripts/tomlparser.rb

```diff
@@ -322,6 +323,16 @@ def populateSettingValuesFromConfigMap(parsedConfig)
     rescue => errorStr
       ConfigParseErrorLogger.logError("config::error: Exception while reading config map settings for annotation based log filtering - #{errorStr}, please check config map for errors")
     end
+
+    #Get Multi-tenancy log collection settings
+    begin
+      if !parsedConfig[:log_collection_settings][:multi_tenancy].nil? && !parsedConfig[:log_collection_settings][:multi_tenancy][:enabled].nil?
```

Comment:
I don't see this added in configmap, will this be added in future stage?  Also should the name follow multi_tenancy_log_collection